Source/Fuse.Elements/Element.Drawing.uno

```diff
@@ -66,21 +66,17 @@ namespace Fuse.Elements
 			if (Visibility != Visibility.Visible)
 				return;
 
+			var visibleRect = GetVisibleViewportInvertPixelRect(dc, RenderBoundsWithEffects);
+			if (visibleRect.Size.X == 0 || visibleRect.Size.Y == 0)
+				return;
+
 			extern double t;
 			if defined(FUSELIBS_PROFILING)
 			{
 				t = Uno.Diagnostics.Clock.GetSeconds();
 				Fuse.Profiling.BeginRegion(this.ToString());
 			}
 
-			var visibleRect = GetVisibleViewportInvertPixelRect(dc, RenderBoundsWithEffects);
-			if (visibleRect.Size.X == 0 || visibleRect.Size.Y == 0)
-			{
-				if defined(FUSELIBS_PROFILING)
-					Fuse.Profiling.EndRegion(Uno.Diagnostics.Clock.GetSeconds() - t);
-				return;
-			}
-
```

Comment:
If you consider doing something like reporting the cull-time as an event if it took a lot of time, you might get the best of both worlds, by the way. As is, this is IMO an move to the worse. Which is sad, because it's holding back otherwise very nice changes. How about you split this part out into a separate PR?